File: 3-strings/1168.cpp
#include <iostream>

using namespace std;

int main(void) {
    int n, i, q;
    char x[102], *px;

    cin >> n;
    for (i = 0; i < n; ++i) {
        cin >> x;
        q = 0;
        px = x;
        while (*px != '\0') {
            switch (*px) {
                case '0': q += 6; break;
                case '1': q += 2; break;
                case '2': q += 5; break;
                case '3': q += 5; break;
                case '4': q += 4; break;
                case '5': q += 5; break;
                case '6': q += 6; break;
                case '7': q += 3; break;
                case '8': q += 7; break;
                case '9': q += 6; break;
            }
            ++px;
        }
        cout << q << " leds"<< endl;
    }
    return 0;
}

File: 3-strings/1168.py
led = {'0': 6, '1': 2, '2': 5, '3': 5, '4': 4,
       '5': 5, '6': 6, '7': 3, '8': 7, '9': 6}
for g in range(int(input())):
    n = input()
    s = 0
    for i in n: s += led[i]
    print(s, 'leds')
